### DISCUSSION TITLE
keyball44で左右自動判別がうまくきいておらず、左にUSBケーブルを刺すと、左側が右と認識されてしまう

### DISCUSSION THREAD

**Discussion by paihu** [Yowkees/keyball#413] [Q&A] [answered]
キーはすべて正常に反応していて、
左のみ、LEFTにはんだを盛っているのですが、

左にUSBケーブルを刺すと、左側が右と認識されてしまって困っています。

どのあたりにミスがありそうか教えていただけると幸いです。
firmwareは Keyball44_via_v1.2.0を利用しています。

![PXL_20231105_063736124 PORTRAIT ORIGINAL](https://github.com/Yowkees/keyball/assets/13479783/09389b3a-d2b3-4315-aaed-e4a41ed1c85a)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

写真を見ると問題なさそうに見えますが、年のため緑丸をつけた２か所の半田付けをやり直してみてください。接触不良があるのかも知れません。また〇をつけたダイオードを新しいものに交換してみていただければ幸いです。

以上、お手数ですがご対応お願いいたします。

![KB44_LEFTjumperfix](https://github.com/Yowkees/keyball/assets/84209772/0b9664b0-20df-4105-b3bf-fed296cc02f7)

**Reply by paihu** [reply]
ご確認ありがとうございます。
上記試してみます

**Reply by paihu** [reply]
ダイオードを新しいものに変えたところ、正常に左右認識するようになりました。

ありがとうございました。

**Comment by Yowkees** [comment]
無事に左右判定されるようになったようで良かったです。安心しました。
ぜひ末永くKeyballをご愛用いただければ幸いです。
